### DISCUSSION TITLE
Passendes Gehäuse für OpenDTU mit Jumper Kabeln

### DISCUSSION THREAD

**Discussion by NintendoTito** [tbnobody/OpenDTU#876] [Q&A]
Hallo zusammen,

ich nutze eine Opendtu aus folgenden Komponenten

> ESP32-DevKitC Core Board ESP32 Entwicklung Bord ESP32-WROOM-32D ESP32-WROOM-32U
> NRF24L01 + PA + LNA Wireless-modul mit Antenne
> 40PCS TZT Dupont Linie – female to female
> TZT ROHS Zertifizierung 0,96 inch oled IIC Serien Weiß OLED Display Modul 128X64

Ans zusammenlöten traue ich mich nicht heran, zudem fehlt mir dazu auch die passende Hardware.

Kann mir jemand ein Case empfehlen, welches von der Größe her so ausgelegt ist, dass ich die Komponenten problemlos unter bekomme?

**Comment by NintendoTito** [comment]
Niemand? 🤔

**Comment by huckes** [comment]
Wenn Deine ästethischen Anspruche nicht zu hoch sind, dann z.Bspl. mit einer Verteilerdose aus dem Baumarkt. Da klebst Du die Teile einfach rein und für die Antenne und das Display bohrst/schneidest Du ensprechende Öffnungen rein. Den Kabelwust sieht dann keiner.
Dann gibt es auch 'ne hübsche Lösung im [Konservenglas](https://blog.helmutkarger.de/balkonkraftwerk-teil-8-opendtu-und-ahoydtu-fuer-hoymiles-wechselrichter/). Da kannst Du auch Die Einzelkomponenten mit Dupon-Kabeln reinschlichten. Das sieht dann zwar noch mehr nach Laborversion aus, aber wenn's gefällt.
Für eine dauerhafte Lösung wäre m.E. aber doch eine Platinenlösung sinnvoll. In der Bucht werden immer wieder fertig verlötete Baugruppen angeboten.

**Comment by geraldriess** [comment]
Hier ein Beispiel [https://github.com/tbnobody/OpenDTU/discussions/247](url)
